uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/my-leave/components/my-leave-report_test.js

```diff
@@ -0,0 +1,50 @@
+(function (CRM) {
+  define([
+    'common/angular',
+    'common/angularMocks',
+    'leave-absences/shared/config',
+    'leave-absences/my-leave/app'
+  ], function (angular) {
+    'use strict';
+
+    describe('myLeaveReport', function () {
+      var $compile, $log, $rootScope, component, controller;
+
+      beforeEach(module('leave-absences.templates', 'my-leave'));
+      beforeEach(inject(function (_$compile_, _$log_, _$rootScope_) {
+        $compile = _$compile_;
+        $log = _$log_;
+        $rootScope = _$rootScope_;
+
+        spyOn($log, 'debug');
+
+        compileComponent();
+      }));
+
+      it('is initialized', function () {
+        expect($log.debug).toHaveBeenCalled();
```

Comment:
I'm not sure about this, you are coupling the initialization of a controller to the presence or not of a variable. What if we decide that we don't need that variable anymore? Or if we decide to remove it?

Calling `$log` is a common pattern in all CiviHR extension, that's why i'm using it to check if the component is initialized